src/lib/components/translater.svelte

```diff
@@ -13,20 +13,49 @@ let filename = [];
 let editorTiny;
 let inputHTML;
 let dbjson = [];
+let safeIndex;
+let safeName;
+let safeInput;
 
 // safe input
 const handelClick = async(index, name) => {
-	if (shown[name][index]) safeAtJson(items[index]);	
+	if (shown[name][index]) {
+		if (editorTiny && (tinymce.activeEditor.getContent()).length > 0 ) {
+			message = !message;
```

Comment:
`showMessage` would be a better name for the variable

Reply:
sounds better and is cahnge